publisher-sdk/src/androidTest/java/com/criteo/publisher/CriteoBannerViewIntegrationTest.java

```diff
@@ -49,13 +50,22 @@ public void setup() throws Exception {
 
     runOnMainThreadAndWait(() -> {
       criteoBannerView = new CriteoBannerView(
-          InstrumentationRegistry.getContext(),
+          InstrumentationUtil.getApplication().getApplicationContext(),
           bannerAdUnit);
     });
 
     criteoBannerView.setCriteoBannerAdListener(criteoBannerAdListener);
   }
 
+
+  @Test
+  public void instantiate_GivenContext_InitializeCorrectly() throws Exception {
+    runOnMainThreadAndWait(() -> {
+      CriteoBannerView criteoBannerView = new CriteoBannerView(InstrumentationUtil.getApplication().getApplicationContext(), null);
```

Comment:
Don't you want to test the constructor without the second parameter ?

Reply:
Sure, though it's not related to my commit :)

Reply:
Didn't you add the constructor with only the context param ?

Reply:
Yes and it is the one that is unit tested here.

Reply:
There you have two parameters, the context and the ad unit, isn't it ?

Reply:
Indeed, I stand corrected. Removed.